rlw-examples/attachments/mssql-monitor.ps1

```diff
@@ -0,0 +1,127 @@
+while ($True) {
+  $nowT = [Math]::Floor([decimal](Get-Date(Get-Date).ToUniversalTime()-uformat "%s"))
+  $res =  Get-WmiObject -Query "select Buffercachehitratio from Win32_PerfFormattedData_MSSQLSERVER_SQLServerBufferManager"
+  if ($res) {
+    Write-Host "PUTVAL $Env:COLLECTD_HOSTNAME/sql-server/buffer-cache-hit-ratio interval=$Env:COLLECTD_INTERVAL ${nowT}:$($res.Buffercachehitratio)"
+  }
+
+  $res = Get-WmiObject -Query "select PercentPrivilegedTime, PercentProcessorTime, PercentInterruptTime from Win32_PerfFormattedData_PerfOS_Processor Where Name='0'"
+    if ($res) {
+      Write-Host "PUTVAL $Env:COLLECTD_HOSTNAME/cpu-0/percent_privileged_time_0 interval=$Env:COLLECTD_INTERVAL ${nowT}:$($res.PercentPrivilegedTime)"
+      Write-Host "PUTVAL $Env:COLLECTD_HOSTNAME/cpu-0/percent_processor_time_0 interval=$Env:COLLECTD_INTERVAL ${nowT}:$($res.PercentProcessorTime)"
+      Write-Host "PUTVAL $Env:COLLECTD_HOSTNAME/cpu-0/percent_interrupt_time_0 interval=$Env:COLLECTD_INTERVAL ${nowT}:$($res.PercentInterruptTime)"
```

Comment:
These look like CPU metrics and there seem to be other things like df and memory below as well. Are these things that should really be in the plugin rather than be built into RightLink's monitoring itself? Some of them might already be in it.